### ISSUE TITLE
[Secure Consul Ph. 1] Integrate ready-to-run signal with Consul bootstrapping done

### ISSUE THREAD

**Issue by jim-wang-intel** [edgexfoundry/edgex-go#3160]
# 🚀 Feature Request

### Relevant Package

 This feature request is for `security-bootstrapper`

### Description
 
Once `consul bootstrapper` is done with installation of agent-token, we need a way to signal it is complete and ready to accept the connection for other services.

### Describe the solution you'd like
 
In `security-bootstrapper`, before raising the `ready-to-run` port/semaphore, we need to wait for consul agent token ready signal or port.
The `consul bootstrapper` also should implement a way to short-circuit the whole ACL and agent token bootstrapping to signal agent token ready if it has already done this successfully before (like 2nd time or later).

### Describe alternatives you've considered
 Have you considered any alternative solutions or workarounds?
